cloc/cli.py

```
@@ -0,0 +1,335 @@
+"""
+Command Line Interface for CLOC tool.
+"""
+
+import argparse
+import sys
+import os
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)